api/server/httputils/form.go

```diff
@@ -23,7 +23,7 @@ func BoolValueOrDefault(r *http.Request, k string, d bool) bool {
 	return BoolValue(r, k)
 }
 
-// Int64ValueOrZero parses a form value into a int64 type.
+// Int64ValueOrZero parses a form value into an int64 type.
 // It returns 0 if the parsing fails.
 func Int64ValueOrZero(r *http.Request, k string) int64 {
```

Comment:
It still has to handle the error, but yes, I think it would be good to have it call the other function for the rest of the logic